components/camera/server_test.go

```diff
@@ -298,20 +261,14 @@ func TestServer(t *testing.T) {
 		test.That(t, resp.ContentType, test.ShouldEqual, "image/png")
 		test.That(t, resp.Data, test.ShouldResemble, imgBuf.Bytes())
 
-		imageReleasedMu.Lock()
```

Comment:
Is it possible to revert these changes? I don't want to have to figure out how they change the meaning of the tests.